ingestion/src/_openmetadata_testutils/factories/metadata/generated/schema/api/classification/create_classification.py

```diff
@@ -0,0 +1,34 @@
+import factory.fuzzy
+
+from _openmetadata_testutils.factories.base.root_model import RootSubFactory
+from _openmetadata_testutils.factories.metadata.generated.schema.type.basic import (
+    MarkdownFactory,
+)
+from metadata.generated.schema.api.classification.createClassification import (
+    AutoClassificationConfig,
+    ConflictResolution,
+    CreateClassificationRequest,
+)
+from metadata.generated.schema.type.basic import EntityName
+
+
+class AutoClassificationConfigFactory(factory.Factory):
```

Comment:
do we want/need these classes under `generated/`? should we find another place for them?